src/features/users.ts

```diff
@@ -0,0 +1,22 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { User } from '../types/User';
+import { getUsers } from '../api/users';
+
+export const loadUsers = createAsyncThunk('users/fetch', async () => {
+  const result = await getUsers().then(usersFromServer => usersFromServer);
+
+  return result;
+});
+
+export const usersSlice = createSlice({
+  name: 'users',
+  initialState: [] as User[],
```

Comment:
It is a good practice to define the type for the initial state of your slice. You can create an interface or type that represents the state of your slice. For example: `type UsersState = User[];` and then use it: `initialState: [] as UsersState`.